modules/service-deployment/regional.tf

```diff
@@ -50,7 +50,7 @@ module "region" {
   ram = {
     create_lag_shares  = local.create_lag_shares
     lag_share_name     = local.lag_share_name
-    target_account_ids = try(flatten(values(data.aws_organizations_organizational_unit_descendant_accounts.target_accounts)[*].accounts[*].id), [])
+    target_account_ids = try([for i in flatten(values(data.aws_organizations_organizational_unit_descendant_accounts.target_accounts)[*].accounts[*]) : i.id if i.state == "ACTIVE"], [])
```

Comment:
1. Should we rename `i` to make it more descriptive?
1. this is quite difficult to read, maybe we could save `values(data.aws_organizations_organizational_unit_descendant_accounts.target_accounts)` to a local and maybe the `*].accounts[*]` part afterwards?